structlog/structlog.go

```diff
@@ -0,0 +1,104 @@
+package structlog
+
+import (
+	"fmt"
+	"reflect"
+	"regexp"
+	"strings"
+
+	"github.com/golang/glog"
+)
+
+var mapregex *regexp.Regexp
+var blacklistregexp []*regexp.Regexp
+
+func init() {
+	mapregex = regexp.MustCompile(`mapstructure:"([^"]+)"`)
+	blacklistregexp = make([]*regexp.Regexp, 1, 1)
+	blacklistregexp[0] = regexp.MustCompile("password")
+}
+
+func LogGeneral(v reflect.Value, prefix string) {
+	logGeneralWithLogger(v, prefix, glog.Infof)
+}
+
+func logGeneralWithLogger(v reflect.Value, prefix string, logger func(msg string, args ...interface{})) {
+	switch v.Kind() {
+	case reflect.Struct:
+		logStructWithLogger(v, prefix, logger)
+	case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64:
+		logInt(v, prefix, logger)
```

Comment:
I did it that way to resemble the struct case and be more easily extensible if we want further functionality in the future. I was initially expecting to need more than just one short line though, so if you really think it would be better inlined, I don't have a problem with that.